kolibri/plugins/learn/assets/src/views/HybridLearningContentCard/index.vue

```diff
@@ -138,14 +138,6 @@
         }
         return styles;
       },
-      maxTitleHeight() {
```

Comment:
The original implementation didn't work in some cases because `this.subtitle` was always `undefined`. I am not sure what was supposed to be represented by `subtitle`. Looking into designs, maybe it was meant to be a category text? (e.g. _"Logical and critical thinking"_ on the screenshot below). In any case, I haven't noticed any indications in the current code that I should maintain this conditional logic because it seems that we don't display a category yet, so I removed it all and just replaced it with the five lines limit for now.

![design](https://user-images.githubusercontent.com/13509191/142989798-f86f0592-0967-4f77-a047-a32a7ffc63fb.png)

Reply:
makes sense to me. I don't know the details of the designs though